src/VisualStudio/CSharp/Test/PersistentStorage/AbstractPersistentStorageTests.cs

```diff
@@ -52,7 +51,6 @@ private void CleanUpPersistentFolder()
         {
         }
 
-        [Fact]
```

Comment:
@Pilchie @jasonmalinowski This is what i see when i try:

![image](https://cloud.githubusercontent.com/assets/4564579/24581964/70e72148-16da-11e7-8758-974eb279035d.png)